src/SFA.DAS.PensionsRegulatorApi.UnitTests/Application/Queries/GetOrganisations/Given_Repository_Throws_An_Exception.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using NSubstitute;
+using NUnit.Framework;
+using FluentAssertions;
+using MediatR;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Data;
+using PensionsRegulatorApi.Domain;
+
+namespace SFA.DAS.PensionsRegulatorApi.UnitTests.Application.Queries.GetOrganisations
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    public class Given_Repository_Throws_An_Exception
+    {
+        private OrganisationRepository _mockRepository;
+        private IRequestHandler<global::PensionsRegulatorApi.Application.Queries.GetOrganisations, IEnumerable<Organisation>> _sut;
+
+        [SetUp]
+        public void Given()
+        {
+            _mockRepository = Substitute.For<OrganisationRepository>();
+
+            _mockRepository
+                .GetOrganisationsForPAYEReference(String.Empty)
+                .ReturnsForAnyArgs(m => throw new Exception());
```

Comment:
.GetOrganisationsForPAYEReference(Arg.Any<string>())
.Throws(new Exception());

looks cleaner imo

Reply:
It does, I agree :)

that test might change now though, if we want the exception to bubble up.